datastore/snippets/snippet_test.go

```diff
@@ -76,7 +76,7 @@ func SnippetNewKey_withMultipleParents() {
 
 func SnippetClient_Put() {
 	ctx := context.Background()
-	client, _ := datastore.NewClient(ctx, "my-proj")
+	client, _ := datastore.NewClientWithDatabase(ctx, "my-proj", "my-database-id")
```

Comment:
issue: use default for tests. Be sure to add a comment that states that the default DB is being used.

Reply:
Here and below.